ihatemoney/static/css/main.css

```diff
@@ -311,8 +311,9 @@ footer .footer-left {
   background: url("../images/see.png") no-repeat right;
 }
 
-#bill_table {
+#bill_table, #month_table {
```

Comment:
I believe we can improve a bit the naming of this table. what about "monthly_expenses_table" ?

Reply:
or `monthly_stats` as you named it like this elsewhere (I find the naming even better than my proposal)

Reply:
Changed.